Leaderboard: setup project

Yazino12 opened the pull request:
### In this pull:

- Add form functionality
- Implement add and refresh functionality

Yazino12 committed:
Implement project basic setup
Add form functionality
Implement add and refresh functionality

BenMukebo commented on dist/index.html:
```diff
@@ -6,5 +6,45 @@
+    <header>
+      <h1>Leaderboard</h1>
+    </header>
+    <main>
+      <div class="left">
+        <div class="header">
+          <h3>Recent scores</h3>
+          <a class="refresh">Refresh</a>
+        </div>
+        <ul class="list"></ul>
+      </div>
+      <div class="right">
+        <h3>Add your score</h3>
+        <form class="add" method="post" action="/">
+          <div>
+            <input
+              name="name"
+              id="name"
+              type="text"
+              maxlength="30"
+              placeholder="Your name"
+            />
+            <label for="name" hidden></label>
+            <input
+              name="score"
+              id="score"
+              type="number"
+              min="0"
```
- [x] Kindly, the HTML code should looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this project.

Your Implementation | Figma Design
:---:|:---:
![leadbad](https://user-images.githubusercontent.com/79058364/161972717-e010b65d-04cd-4a5b-99e4-047041d4d3e1.png)| ![lead](https://user-images.githubusercontent.com/79058364/161972836-02bc3c3a-cda8-4b56-a6cd-e1fb24bfe9ff.png)

Yazino12 replied:
Hello @BenMukebo, thanks for taking the time. The way I implemented this is that you can add scores, so I took the hardcoded HTML completely, and if you add some scores through the form, scores will appear in the list.
Should I add a hardcoded score list in the HTML as well?

BenMukebo replied:
Yes please do it because you are not working on add scores yet by making a POST, GET request to the API, as per the requirement of this first step you just need to display the HTML without no API functionality

BenMukebo reviewed (changes requested):
### Hi @Yazino12,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Please, Be use to make more commit, Commitment helps you stick to your goals during the good times and the bad times Because when things really tend to go south the revision history is a great resource to find out quickly where exactly things went wrong

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Yazino12 committed:
Apply required changes
Add initial starting data to the list
Modify list item background according to listing

ShadyShawkat reviewed (approved):
## Hi @Yazino12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Image](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
